packages/pn-pa-webapp/src/api/interceptors.ts

```diff
@@ -1,16 +1,74 @@
-import { EnhancedStore } from "@reduxjs/toolkit";
-import { apiClient } from "./apiClients";
+import { EnhancedStore } from '@reduxjs/toolkit';
+// import { NotificationStatus } from '@pagopa-pn/pn-commons';
+import { NotificationDetail, NotificationStatus, TimelineCategory } from '@pagopa-pn/pn-commons';
+import { apiClient } from './apiClients';
 
 export const setUpInterceptor = (store: EnhancedStore) => {
   apiClient.interceptors.request.use(
     (config) => {
-        /* eslint-disable functional/immutable-data */
-        const token: string = store.getState().userState.user.sessionToken;
-        if (token && config.headers) {
-          config.headers.Authorization = 'Bearer ' + token;
-        }
-        return config;
-      },
-      (error) => Promise.reject(error)
-    );
-};
\ No newline at end of file
+      /* eslint-disable functional/immutable-data */
+      const token: string = store.getState().userState.user.sessionToken;
+      if (token && config.headers) {
+        config.headers.Authorization = 'Bearer ' + token;
+      }
+      return config;
+    },
+    (error) => Promise.reject(error)
+  );
+
+  apiClient.interceptors.response.use(
+    (response) => {
```

Comment:
they are missed the cases:
- list of notifications
- all the cases of the cancelled api (success, already cancelled, generic error)